### PR TITLE
Return of Influence DCP operators

### PR DESCRIPTION
**Pull request by ViniciusCampanha**
Influence DCP operators returns the list of DCP alias.

### CODE REVIEW

File: src/terrama2/services/analysis/core/dcp/influence/PythonOperator.cpp [hunk 1]
```diff
@@ -36,24 +36,24 @@ boost::python::list terrama2::services::analysis::core::dcp::zonal::influence::p
   std::vector<std::string> vecAttr;
   terrama2::services::analysis::core::python::pythonToVector<std::string>(attributeList, vecAttr);
 
-  auto vecIds = terrama2::services::analysis::core::dcp::zonal::influence::byAttribute(dataSeriesName, vecAttr);
+  auto vecDCP = terrama2::services::analysis::core::dcp::zonal::influence::byAttribute(dataSeriesName, vecAttr);
 
   boost::python::list pyList;
-  for(DataSetId id : vecIds)
+  for(auto& dcp : vecDCP)
```
Comment (janosimas) [inline src/terrama2/services/analysis/core/dcp/influence/PythonOperator.cpp]: use const if not changing 'dcp'

File: src/terrama2/services/analysis/core/dcp/influence/PythonOperator.cpp [hunk 2]
```diff
@@ -36,24 +36,24 @@ boost::python::list terrama2::services::analysis::core::dcp::zonal::influence::p
...
 {
-  auto vecIds = terrama2::services::analysis::core::dcp::zonal::influence::byRule(dataSeriesName, buffer);
+  auto vecDCP = terrama2::services::analysis::core::dcp::zonal::influence::byRule(dataSeriesName, buffer);
 
   boost::python::list pyList;
-  for(DataSetId id : vecIds)
+  for(auto& dcp : vecDCP)
   {
-    pyList.append(boost::python::object(id));
+    pyList.append(boost::python::object(dcp));
```
Comment (janosimas) [inline src/terrama2/services/analysis/core/dcp/influence/PythonOperator.cpp]: same as last comment

File: webapp/public/javascripts/angular/data-series/registration.js [hunk 3]
```diff
@@ -665,9 +677,12 @@ angular.module('terrama2.dataseries.registration', [
         };
 
         var dataSeriesType = dataSeries.data_series_semantics.data_series_type_name;
-        
-        $scope.dataSeriesGroups[0].children = _helper($scope.dataSeriesGroups[0].children);
-        
+        if (dataSeriesType === globals.enums.DataSeriesType.GRID) {
```
Comment (franciscovn) [inline webapp/public/javascripts/angular/data-series/registration.js]: @ViniciusCampanha This code cannot be changed

File: webapp/public/javascripts/angular/data-series/registration.js [hunk 4]
```diff
@@ -629,8 +639,10 @@ angular.module('terrama2.dataseries.registration', [
 
           if (ds.data_series_semantics.data_series_type_name === globals.enums.DataSeriesType.GRID) {
             ds.isGrid = true;
+            _helper(1, ds);
           } else {
             ds.isGrid = false;
+            _helper(0, ds);
```
Comment (franciscovn) [inline webapp/public/javascripts/angular/data-series/registration.js]: @ViniciusCampanha This code cannot be changed

File: webapp/public/javascripts/angular/data-series/registration.js [hunk 5]
```diff
@@ -629,8 +639,10 @@ angular.module('terrama2.dataseries.registration', [
 
           if (ds.data_series_semantics.data_series_type_name === globals.enums.DataSeriesType.GRID) {
             ds.isGrid = true;
+            _helper(1, ds);
```
Comment (franciscovn) [inline webapp/public/javascripts/angular/data-series/registration.js]: @ViniciusCampanha This code cannot be changed

File: src/terrama2/services/analysis/core/AnalysisExecutor.cpp [hunk 6]
```diff
@@ -385,27 +385,40 @@ void terrama2::services::analysis::core::AnalysisExecutor::storeMonitoredObjectA
     throw terrama2::InvalidArgumentException() << ErrorDescription(errMsg);
   }
 
-  auto dataSeries = dataManager->findDataSeries(analysis->outputDataSeriesId);
+  auto outputDataSeries = dataManager->findDataSeries(analysis->outputDataSeriesId);
```
Comment (ViniciusCampanha) [inline src/terrama2/services/analysis/core/AnalysisExecutor.cpp]: Removed

File: src/terrama2/services/analysis/core/AnalysisExecutor.cpp [hunk 7]
```diff
@@ -385,27 +385,40 @@ void terrama2::services::analysis::core::AnalysisExecutor::storeMonitoredObjectA
...
-  if(!dataSeries)
+  if(!outputDataSeries)
   {
     QString errMsg = QObject::tr("Could not find the output data series.");
     throw terrama2::InvalidArgumentException() << ErrorDescription(errMsg);
   }
 
-  auto dataProvider = dataManager->findDataProvider(dataSeries->dataProviderId);
+  auto dataProvider = dataManager->findDataProvider(outputDataSeries->dataProviderId);
   if(!dataProvider)
```
Comment (ViniciusCampanha) [inline src/terrama2/services/analysis/core/AnalysisExecutor.cpp]: Removed

**Pull request merged**